docs/rules/no-async-before.md

```diff
@@ -0,0 +1,52 @@
+# Prevent using async/await in Cypress test cases (no-async-tests)
+
+Cypress before that return a promise will cause tests to prematurely start, can cause unexpected behavior. An `async` function returns a promise under the hood, so a before using an `async` function might also error.
+
+## Rule Details
+
+This rule disallows using `async` before and beforeEach functions.
```

Comment:
```suggestion
This rule disallows using `async` `before` and `beforeEach` functions.
```

Reply:
```suggestion
This rule disallows using `async` in `before` and `beforeEach` hook functions.
```

Sorry, @AtofStryker , just a slight update to the suggestion :)